p4-16/psa/PSA.mdk

```diff
@@ -985,7 +982,7 @@ contents of `packet_in` of that ingress packet, with no modifications
 from any ingress processing.  Truncation of the payload is supported.
 
 Packets cloned in ingress using a clone session configured with
-`clone_port` equal to `PSA_PORT_RECIRCULATE` fit into this category.
+`egress_port` equal to `PSA_PORT_RECIRCULATE` fit into this category.
```

Comment:
`clone_port` is an obsolete name that was formerly in the PSA as a configuration value for a clone session.  Updating it.